### PR TITLE
Missing path

### PR DESCRIPTION
**Pull request by jburel**
Update the documentation section
and also check the setting in CI

### CODE REVIEW

File: ansible/templates/common/gunicorn.j2 [hunk 1]
```diff
@@ -13,4 +13,14 @@ Additional settings can be configured by changing the following properties:
...
 
-{% endif %}
\ No newline at end of file
+{% else %}
+#!/bin/bash
+
+set -e -u -x
+export PATH=/opt/omero/web/venv3/bin:$PATH
+
+omero config set omero.web.application_server.max_requests 500
```
Comment (manics) [inline ansible/templates/common/gunicorn.j2]: Is this needed?

Comment (jburel) [reply ansible/templates/common/gunicorn.j2]: This is only to apply what we describe in the doc

Comment (manics) [reply ansible/templates/common/gunicorn.j2]: There's no sign of it on any of our prod-playbook servers

Comment (jburel) [reply ansible/templates/common/gunicorn.j2]: In that case we could remove that from the doc

**Comment by joshmoore** [comment]
What's the next step here?

**Comment by jburel** [comment]
The corresponding doc PR https://github.com/ome/ome-documentation/pull/2105 has been approved.
Merging this PR so the settings will not appear in the step-by-step guide

**Pull request merged**